app/slashcommands/command_loadtest.go

```diff
@@ -289,25 +310,40 @@ func (*LoadTestProvider) UsersCommand(a *app.App, c request.CTX, args *model.Com
 	cmd := strings.TrimSpace(strings.TrimPrefix(message, "users"))
 
 	doFuzz := false
-	if strings.Index(cmd, "fuzz") == 0 {
+	if fuzzRE.MatchString(cmd) {
 		doFuzz = true
-		cmd = strings.TrimSpace(strings.TrimPrefix(cmd, "fuzz"))
 	}
 
-	usersr, ok := parseRange(cmd, "")
-	if !ok {
-		usersr = utils.Range{Begin: 2, End: 5}
+	var err error
+	rng := utils.Range{Begin: 2, End: 5}
+	rangeParam := getMatch(rangeRE, cmd)
+	if rangeParam != "" {
+		rng, err = parseRange(rangeParam)
+		if err != nil {
+			return &model.CommandResponse{Text: "Failed to add users: " + err.Error(), ResponseType: model.CommandResponseTypeEphemeral}, err
+		}
+
 	}
 
 	team, err := a.Srv().Store().Team().Get(args.TeamId)
 	if err != nil {
 		return &model.CommandResponse{Text: "Failed to add users", ResponseType: model.CommandResponseTypeEphemeral}, err
 	}
 
+	time := int64(0)
+	timeParam := getMatch(timeRE, cmd)
+	if timeParam != "" {
+		time, err = strconv.ParseInt(timeParam, 10, 64)
```

Comment:
What about using this for some added validation?
```suggestion
		time, err = strconv.ParseUint(timeParam, 10, 64)
```